src/Shared/TaskHostConfiguration.cs

```diff
@@ -182,6 +187,7 @@ internal class TaskHostConfiguration : INodePacket
             _culture = culture;
             _uiCulture = uiCulture;
 #if FEATURE_APPDOMAIN
+            _appDomainConfigBytes = appDomainSetup?.GetConfigurationBytes();
             _appDomainSetup = appDomainSetup;
```

Comment:
Is this still needed?
If yes - how is it used

Reply:
When you create a new instance of the type (not by deserialization), then we need to set the byte[] for serialization.

Reply:
Sorry - the comment was assigned to wrong line (mobile tiny UX :-)). It was meant for `_appDomainSetup` - that one should not be needed anywhere.

Reply:
I had offline discussion with Ladi. We agreed that extracting config bytes should be done just before the serialization. I modified the code but we need reference to the AppDomainSetup.